src/mobile/Footer/Footer.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../scss/variables.scss";
+
+.footer {
+  background: $primaryColorLight;
+  padding: 48px 67px 24px;
+  
+  .footerContent {
+    margin: 0 auto;
+    max-width: 1380px;
```

Comment:
We could move it to const in variables.scss

Reply:
We can use `width: 100%`, the corresponding width will be set in the wrapper in the nextjs project

Reply:
Let's discuss sizing of elements also in person or by phone.
It would be cool if we could set only width or height of an element and other sizes of element (paddings, margins, font-size, ...) will be calculated based on one base value. In this way component will be truly independent and isolated.